### PR TITLE
update easyMock to mock in chime test

### LINKED ISSUE #237: [Testing] Remove Easymock and replace with mockk framework

### PR DESCRIPTION
**Pull request by rohan19a**
### Description
Updated a chime webhook test from easyMock to Mockk

### Issues Resolved
#237

### Check List
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: notifications/core/src/test/kotlin/org/opensearch/notifications/core/destinations/ChimeDestinationTests.kt [hunk 1]
```diff
@@ -87,23 +87,22 @@ internal class ChimeDestinationTests {
...
 
         val httpClient = DestinationHttpClient(mockHttpClient)
         val webhookDestinationTransport = WebhookDestinationTransport(httpClient)
         DestinationTransportProvider.destinationTransportMap = mapOf(DestinationType.CHIME to webhookDestinationTransport)
 
         val title = "test Chime"
         val messageText = "{\"Content\":\"Message gughjhjlkh Body emoji test: :) :+1: " +
-            "link test: http://sample.com email test: marymajor@example.com All member call out: " +
-            "@All All Present member call out: @Present\"}"
+                "link test: http://sample.com email test: marymajor@example.com All member call out: " +
```
Comment (yuye-aws) [inline notifications/core/src/test/kotlin/org/opensearch/notifications/core/destinations/ChimeDestinationTests.kt:104]: Is this change intended?

**Comment by yuye-aws** [comment]
Command `./gradlew run` cannot run successfully on the main branch. Perhaps this is why workflow tests failed now. Please rerun these tests when `./gradlew run` issue gets fixed.

**Comment by Hailong-am** [comment]
@rohan19a `ktlint` check has failed. can you run `./gradlew ktlintFormat` to fix it?

**Comment by yuye-aws** [comment]
Some tests get failed. Please take a look at why.

**Review by Hailong-am — DISMISSED** [review]
LGTM

**Comment by zhichao-aws** [comment]
Can we replace the EasyMock with mockk in this [test case](https://github.com/opensearch-project/notifications/blob/62b7b4f840e7e224f947f1745da375df6f419b9b/notifications/core/src/test/kotlin/org/opensearch/notifications/core/destinations/ChimeDestinationTests.kt#L121) also? Then we can remove the dependency for EasyMock in ChimeDestinationTests

**Comment by rohan19a** [comment]
> Can we replace the EasyMock with mockk in this [test case](https://github.com/opensearch-project/notifications/blob/62b7b4f840e7e224f947f1745da375df6f419b9b/notifications/core/src/test/kotlin/org/opensearch/notifications/core/destinations/ChimeDestinationTests.kt#L121) also? Then we can remove the dependency for EasyMock in ChimeDestinationTests

Missed that somehow, I'll edit that and recommit

**Review by zhichao-aws — APPROVED** [review]
lgtm

**Review by Hailong-am — APPROVED** [review]
Looks good to me, we have serval files still using easyMock, we can have follow up PR to remove them all, after that we can remove the dependency of easyMock

**Comment by yuye-aws** [comment]
Shall we manually backport this PR to 2.x branch?

**Comment by Hailong-am** [comment]
@rohan19a would you mind to do manual backport according to https://github.com/opensearch-project/notifications/pull/815#issuecomment-1815701667

**Comment by zhichao-aws** [comment]
In main branch we use mockk to create a CloseableHttpResponse instance, while in 2.x we still used EasyMock, and the code inconsistence causes the backport fail. We need to manual backport and replace all EasyMock usage in the backport PR.
